packages/@webex/plugin-meetings/test/unit/spec/meeting/index.js

```diff
@@ -677,14 +741,68 @@ describe('plugin-meetings', () => {
           }
         });
       });
-      describe('#stopReceivingTranscription', () => {
-        it('should get invoked', () => {
-          meeting.transcription = {
-            closeSocket: sinon.stub(),
-          };
 
-          meeting.stopReceivingTranscription();
-          assert.calledOnce(meeting.transcription.closeSocket);
+      describe('#stopTranscription', () => {
+        beforeEach(() => {
+          webex.internal.voicea.on = sinon.stub();
+          webex.internal.voicea.off = sinon.stub();
+          webex.internal.voicea.listenToEvents = sinon.stub();
+          webex.internal.voicea.toggleTranscribing = sinon.stub();
+        });
+        it('should stop listening to voicea events and also trigger a stop event', () => {
```

Comment:
can we leave a blank line before this test?

Reply:
Added